dashboard/package.json

```diff
@@ -26,6 +26,7 @@
     "watch-css": "yarn run build-css && yarn run compile-lang && sass src/ --watch --no-source-map"
   },
   "dependencies": {
+    "@babel/parser": "^7.15.3",
```

Comment:
I don't think this one is required. Sometimes it becomes added when doing `yarn upgrade`, but it is already present as part of the react-scripts deps. If you remove it and then run yarn install, you'll notice everything works fine.

Reply:
Done!